Modules/_xxsubinterpretersmodule.c

```diff
@@ -627,6 +627,30 @@ _channelends_associate(_channelends *ends, int64_t interp, int send)
     return 0;
 }
 
+static int64_t *
+_channelends_list_interpreters(_channelends *ends, int64_t *count, int send)
+{
+    int64_t *ids = NULL;
+    int64_t numopen = send ? ends->numsendopen : ends->numrecvopen;
+    if (numopen >= PY_SSIZE_T_MAX) {
```

Comment:
numopen is an `int64_t`, not a `ssize_t`

Reply:
Have added a `PY_INT64_T_MAX` macro (and related macros).